src/Event/Listener/ContentFillListener.php

```diff
@@ -32,25 +30,6 @@ public function __construct(Config $config, ContentExtension $contentExtension,
         $this->users = $users;
     }
 
-    public function preUpdate(LifecycleEventArgs $args): void
```

Comment:
Should we leave this method in place, as an empty placeholder, with an `@deprecated`? Then it won't break for people who haven't updated their `services.yaml` yet.

Reply:
yes, I'll put that back.